examples/proc_nutation.py

```diff
@@ -0,0 +1,94 @@
+"""
+Process nutation data
+====================
+`py proc_nutation.py NODENAME FILENAME EXP_TYPE`
+
+Fourier transforms (and any needed data corrections for older data) are performed according to the `postproc_type` attribute of the data node.
+This script plots the result, as well as signal that's averaged along the `nScans` dimension.
+
+Tested with:
+
+``py proc_nutation.py nutation_2 240710_27mM_TEMPOL_chokes_T_atprobe_nutation.h5 ODNP_NMR_comp/nutation``
+
+"""
+from pyspecProcScripts.load_data import lookup_table
+from pyspecProcScripts import select_pathway, determine_sign, fid_from_echo, zeroth_order_ph
+from pyspecdata import *
+import numpy as np
+import sys
+import os
+from itertools import cycle
+
+colorcyc_list = plt.rcParams["axes.prop_cycle"].by_key()["color"]
+colorcyc = cycle(colorcyc_list)
+
+assert len(sys.argv) == 4
+d = find_file(
+    sys.argv[2], exp_type=sys.argv[3], expno=sys.argv[1], lookup=lookup_table
+)
+if d.get_prop("postproc_type") == "spincore_SE_v1":
+    d.rename("indirect", "p_90")
+    d["p_90"] *= 1e-9
+elif "indirect" in d.dimlabels:
+    d.rename("indirect","p_90")
+with figlist_var() as fl:
+    # {{{ Manual processing
+    d.squeeze()
+    d.ift('t2')
+    d['t2'] -= 2e-3 # eyeball correction
+    d.ft('t2')
+    fl.next("Raw Data")
+    fl.image(d, interpolation="auto")
+    fl.next("raw data, phase")
+    ph0 = select_pathway(d)['t2':(-100,0)]['p_90',:5].sum('t2').sum('p_90')
+    ph0 /= abs(ph0)
```

Comment:
why are we committing this very manual correction? This doesn't make sense.

Reply:
this was your og version from the original PR I have since edited to what I have actually been using to process nutations